packages/block-library/src/navigation/edit.js

```diff
@@ -78,9 +78,20 @@ function Navigation( {
 		}
 	);
 
+	const blockClassNames = classnames( className, {
```

Comment:
Nit: As far as I understand `useBlockProps`, you should pass `blockClassNames` as `className` there and you don't have to worry about the class name that block wrapper defines.

Reply:
Good suggestion, moved classnames to useBlockProps in a97e8f2874